src/Ydb.Sdk/tests/Value/TestBasicUnit.cs

```diff
@@ -256,6 +256,38 @@ public void OptionalPrimitiveCast()
             Assert.Null((TimeSpan?)(YdbValue)(TimeSpan?)null);
         }
 
+        [Fact]
+        public void DecimalType()
```

Comment:
Need integration tests:
1. Select same value from server
2. Insert decimal values to table